front/app/containers/DisabledAccount/messages.ts

```diff
@@ -0,0 +1,17 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  title: {
+    id: 'app.containers.DisabledAccount.title',
+    defaultMessage: 'Your account has been temporarily disabled',
+  },
+  text: {
+    id: 'app.containers.DisabledAccount.text',
+    defaultMessage:
+      'Your account on Greenville Participation Platform has been temporarily disabled for a violation of the community guideline. For more information on this, you can consult the Terms & conditions.',
```

Comment:
Greenville Participation Platform

This needs to change to a variable. "the participation platform of {organizationName}". Doble check.

Reply:
Terms & conditions => terms & conditions